src/playbacks/hls/hls.js

```diff
@@ -28,12 +29,73 @@ export default class HLS extends HTML5VideoPlayback {
     this._hls.currentLevel = this._currentLevel
   }
 
+  get _duration() {
+    return this._playableRegionDuration
+  }
+
+  get _startTime() {
+    if (this._playbackType === Playback.LIVE && this._playlistType !== "EVENT") {
+      return this._extrapolatedStartTime
+    }
+    return this._playableRegionStartTime
+  }
+
+  get _now() {
```

Comment:
@leandromoreira perhaps we should move this to utils. It could be useful for other components.